modules/exploits/unix/webapp/mybb_backdoor.rb

```diff
@@ -24,7 +24,7 @@ def initialize(info = {})
         [
           [ 'OSVDB', '76111' ],
           [ 'BID', '49993' ],
-          [ 'SECUNIA', '46300' ],
```

Comment:
This is the only module that uses this format, updated to the manually sourced web archive URL